src/rpc/RPC.cpp

```diff
@@ -30,7 +66,7 @@ make_WsContext(
 
     std::string command = commandValue.as_string().c_str();
 
-    return Context{
+    return std::make_optional<Context>(
```

Comment:
why do we need the `make_optional` now, whereas before we did not?

Reply:
It's because we were relying on implicit conversion which relied on copy-constructor of `Context`. Now that `Context` is inheriting from `Taggable` its copy constructor is implicitly deleted since we are using `unique_ptr` deep inside (move only type). `std::make_optional` does not rely on copying anything and just creates `Context` in-place. In any case we should have been using `make_optional` in the first place.